xml/System/Type.xml

```diff
@@ -7672,7 +7669,7 @@ If the current <xref:System.Type> represents a type parameter in the definition
           <format type="text/markdown"><![CDATA[
 
 ## Remarks
- The <xref:System.Type.GetOptionalCustomModifiers%2A> and <xref:System.Type.GetRequiredCustomModifiers%2A> methods are provided for designers of managed compilers. For more information on custom modifiers, see classes in the <xref:System.Runtime.CompilerServices> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).
+ The <xref:System.Type.GetOptionalCustomModifiers%2A> and <xref:System.Type.GetRequiredCustomModifiers%2A> methods are provided for designers of managed compilers. For more information on custom modifiers, see classes in the <see cref="N:System.Runtime.CompilerServices"/> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).
```

Comment:
Same. GitHub does not let me offer a full suggestion to convert the whole markdown section to xml, so we need to revert this:

```suggestion
 The <xref:System.Type.GetOptionalCustomModifiers%2A> and <xref:System.Type.GetRequiredCustomModifiers%2A> methods are provided for designers of managed compilers. For more information on custom modifiers, see classes in the <xref:System.Runtime.CompilerServices> namespace. Also see the metadata specification in Partition II of the [Common Language Infrastructure (CLI) documentation](https://www.ecma-international.org/publications-and-standards/standards/ecma-335/).
```